### PR TITLE
Add literal memory usage stat

### PR DESCRIPTION
**Pull request by hirotnk**

### CODE REVIEW

**Comment by hirotnk** [comment]
Hi @djnym , We recently got VM crash with message "literal_alloc: Cannot allocate 13981176 bytes of memory (of type "literal")". So I'd like to keep track of the usage of it. At first I thought I can use `code` part returned from `erlang:memory/0` API. But when I did some experiment by adding literal data using mochiglobal, that part did not increase while `carriers_size` part of literal_alloc showed expected data size in bytes when it reached to the limit and crashed. So this is why I used `erlang:system_info/1`. For the details of fields from that API, I referenced this doc: http://www.erlang-factory.com/static/upload/media/139454517145429lukaslarsson.pdf

**Comment by hirotnk** [comment]
Looking at the change, I'd like to make a function and add unit test. Will work on it.

File: src/mondemand_vmstats.erl [hunk 1]
```diff
@@ -382,10 +388,51 @@ scheduler_wall_time_diff (PrevSchedWallTime, SchedWallTime) ->
     <- lists:zip(lists:sort(PrevSchedWallTime),lists:sort(SchedWallTime))
   ].
 
+% Sum of current carriersizes(mcbs+scbs) for all instances although
+% currently there is only one global instance for literal_alloc.
+% Note: This function returns constant value zero when OTP version
+% is 18 or before.
+-ifdef (namespaced_types_19_later).
```
Comment (djnym) [inline src/mondemand_vmstats.erl]: I don't like 'namespaced_types_19_later' as that's not what this is.  I believe it's actually something like 'allocator_stats_available', right?  This doesn't have anything to do with types AFAIK.

Comment (hirotnk) [reply src/mondemand_vmstats.erl]: @djnym Thank you, updated.

**Pull request merged**